axum/src/extract/multipart.rs

```diff
@@ -142,6 +142,11 @@ impl<'a> Field<'a> {
     pub async fn text(self) -> Result<String, MultipartError> {
         self.inner.text().await.map_err(MultipartError::from_multer)
     }
+    
+    // Stream a chunk of the field data.
+    pub async fn chunk(self) -> Result<Option<Bytes>, MultipartError> {
```

Comment:
I think this needs to take `&mut self` otherwise you can only access the first chunk.

Reply:
Was debating this as well. Updated it to use `&mut self`